File: 75.cpp
#include"bigint.h"
#include<cmath>
using namespace std;
int main()
{int x=0,size=1500001,*a=new int[size],ans=0;
for(int i=0;i<size;i++)
a[i]=0;
 for(int i=1;i<size;i++)
{
for(int j=i;sqrt(i*i+j*j)<size;j++)
{
  int m=sqrt(i*i+j*j);
  if(m==1.0*sqrt(i*i+j*j))
  if(i+j+m<size)
  {cout<<i<<"\t"<<j<<"\t"<<m<<"\t"<<i+j+m<<endl;a[i+j+m]++;}
}
}
for(int i=0;i<size;i++)
{//cout<<a[i]<<endl;
if(a[i]==1)
ans++;
}
cout<<ans;

  return 0;
}

File: 75.js
var utils = require('./vam54ee.utils.js');

var ans = 0;
for(var n = 12; n < 1500000 ; n+=2)
{
  var count = 0;
  var y = Math.ceil(n/(2+Math.sqrt(2)));
  do
  {
    var z = (2*y*y-2*n*y+n*n)/(2*n-2*y);
    var x = 0;
    if(utils.isInteger(z))
    {
      x = Math.sqrt(z*z - y*y);
      if(x < y && y < z && x+y+z === n)
      {
        count++;
        if(n% 100000 === 0)
        console.log(x,y,z,n);
      }
    }
    y++;
    //console.log(y);
  }while(y + z < n);
  if(count === 1)
    ans++;
}
console.log(ans);